cfgov/scripts/http_smoke_test.py

```diff
@@ -108,37 +121,46 @@
     '/policy-compliance/guidance/implementation-guidance/hmda-implementation/',
     '/policy-compliance/guidance/implementation-guidance/mortserv/',
     '/policy-compliance/guidance/implementation-guidance/tila-respa-disclosure-rule/',  # noqa: E501
-    '/about-us/',
-    '/about-us/the-bureau/',
     '/about-us/budget-strategy/',
     '/about-us/payments-harmed-consumers/',
     '/about-us/blog/',
     '/about-us/newsroom/',
     '/about-us/events/',
-    '/activity-log/',
     '/about-us/careers/',
     '/about-us/careers/current-openings/',
     '/about-us/doing-business-with-us/',
     '/about-us/innovation/',
-    '/about-us/contact-us/',
-    '/eregulations/',
-    '/eregulations/1026',
+    '/activity-log/',
+    '/your-story/',
 ]
 
+FULL_RUN = sorted(set(TOP20 + APPS))
+
+
+def get_full_list():
+    try:
+        url_data = requests.get(S3_URI).json()
+    except Exception as e:
```

Comment:
I was anticipating possible network or s3 outages that might be hard to anticipate, and I think our fallback list of URLs is robust enough that failing to get the S3 list is not crucial.

Since this job can stop deployments, I thought we should err on the side of letting a potentially temporary S3 issue pass rather than stop deployments to debug S3.